### DISCUSSION TITLE
Any plans to make a port of React Icons for SolidJS

### DISCUSSION THREAD

**Discussion by thedanchez** [react-icons/react-icons#689] [General]
Working on migrating a project from React to Solid and it would be great if there was a port of React Icons for SolidJS (so Solid Icons). Does anyone know if there are plans to create one? Man that would be so cool.

Or would it be better to use something like https://github.com/antfu/unplugin-icons instead?

**Comment by kamijin-fanta** [comment]
SolidJS is of great personal interest to me, but I don't have any ideas at this time for adapting this project to other libraries.

I would be willing to consider developing this project if there is sufficient testing, etc. within our community.
